pages/linux/sar.md

```diff
@@ -0,0 +1,27 @@
+# sar
+
+> Monitor performance of various Linux subsystems.
+
+- Report I/O and Transfer rate every 1 second:
```

Comment:
Don't think `Transfer` needs to have a capital letter at the beginning.

Reply:
sure, acutally the manpage had I/O and Transfer in upper-case, so i did the same.
anyways changed